classifides/ads/context_processors.py

```diff
@@ -0,0 +1,6 @@
+from .models import Category
+
+
+def category_processor(request):

```

Comment:
Where is this used ? Please remove this if not used.

Reply:
It is used in the nav bar of all the templates